services/victoriametrics/scrape_configs.go

```diff
@@ -397,34 +397,12 @@ func scrapeConfigsForPostgresExporter(s *models.MetricsResolutions, params *scra
 		return nil, err
 	}
 
-	mrOptions := []string{
-		"custom_query.mr",
-	}
-	mrOptions = collectors.FilterOutCollectors("", mrOptions, params.agent.DisabledCollectors)
-	mr, err := scrapeConfigForStandardExporter("mr", s.MR, params, mrOptions)
-	if err != nil {
-		return nil, err
-	}
-
-	lrOptions := []string{
-		"custom_query.lr",
-	}
-	lrOptions = collectors.FilterOutCollectors("", lrOptions, params.agent.DisabledCollectors)
-	lr, err := scrapeConfigForStandardExporter("lr", s.LR, params, lrOptions)
-	if err != nil {
-		return nil, err
-	}
+	// MR and LR is not supported by postgres_exporter, so skipping them.
```

Comment:
But why figured that out only now?

Reply:
I found it a few month ago, but had no time to fix and test it.